Sources/Fuzzilli/Minimization/BlockReducer.swift

```diff
@@ -82,6 +85,24 @@ struct BlockReducer: Reducer {
         
         verifier.tryNopping(candidates, in: program)
     }
+
+    private func reduceCodeString(codestring: BlockGroup, in program: Program, with verifier: ReductionVerifier){
```

Comment:
I don't think we have a decent mechanism to do this yet, so can you just add a `// TODO write a test for this reduction` here?

Reply:
done